tests/codex/sales/testreservations.nim

```diff
@@ -21,7 +21,7 @@ asyncchecksuite "Reservations module":
     reservations: Reservations
 
   setup:
-    randomize()
+    randomize(1.int64) # create reproducible results
```

Comment:
This is also possible, but it will run with the same inputs every time. If it is not a problem then keep it.